daemon/daemon.go

```diff
@@ -730,7 +736,7 @@ func (daemon *Daemon) Shutdown() error {
 	}
 
 	if daemon.containers != nil {
-		logrus.Debug("starting clean shutdown of all containers...")
+		logrus.Debugf("starting clean shutdown of all containers in %d seconds...", daemon.configStore.ShutdownTimeout)
```

Comment:
Thanks @mlaventure @thaJeztah . I will temporarily settle with `start clean shutdown of all containers with a %d seconds timeout...` but let me know if there are other suggestions.